### DISCUSSION TITLE
suggestion: link to geograph

### DISCUSSION THREAD

**Discussion by backache** [openbenches/openbenches.org#321] [Ideas]
I was wondering whether when someone drills down to a particular bench (in the UK) there should be a link to geograph?

For example, looking at 

https://openbenches.org/bench/17849

would have a link based on the benches' co-ordinates (51.408226,0.013411 ) in case a viewer wants to see more pictures of the area eg :-

https://www.geograph.org.uk/mapper/combined.php#18/51.408226/0.013411

**Comment by edent** [comment]
It's a good idea. But is there a way to see if there *are* any other photos in the area? In the example linked above, there aren't. And I don't want to send people to a site where there's nothing for them to do.
